upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -148,6 +148,7 @@ func NewAWSCloud(region string, tags map[string]string) (AWSCloud, error) {
 		c := &awsCloudImplementation{region: region}
 
 		config := aws.NewConfig().WithRegion(region)
+		//config := aws.NewConfig().WithRegion(region).WithLogLevel(aws.LogDebugWithHTTPBody)
```

Comment:
I am going to add it to documentation in the go file, and file an issue to be able to turn on debugging. I will submit another PR.